source/inverse.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Функция возвращает инвертированный массив, если он без дополнительного параметра.
+ * Если передать положительный параметр n вместе с массивом, 
+ * он инвертирует массив с n-го элемента, с отрицательным -до |n|-го элемента.
+ * @param {array} array инвертируемый массив
+ * @param {number} index индекс, от(до) которого идет инверсия
+ * @returns {array} возвращаемый массив
```

Comment:
```suggestion
 * @param {Array} array инвертируемый массив
 * @param {number} index индекс, от(до) которого идет инверсия
 * @returns {Array} возвращаемый массив
```
https://www.google.com/search?q=jsdoc+any+array -> https://stackoverflow.com/questions/14611995/how-to-specify-an-array-of-objects-as-a-parameter-or-return-value-in-jsdoc